Fixing desktop exec codes.
--- a/eric/eric6.desktop
+++ b/eric/eric6.desktop
@@ -1,7 +1,7 @@
 [Desktop Entry]
 Version=1.0
 Type=Application
-Exec=eric6@MARKER@
+Exec=eric6 %F@MARKER@
 MimeType=text/x-python;
 Icon=eric@MARKER@
 Terminal=false
--- a/eric/eric6_webbrowser.desktop
+++ b/eric/eric6_webbrowser.desktop
@@ -1,7 +1,7 @@
 [Desktop Entry]
 Version=1.0
 Type=Application
-Exec=eric6_webbrowser@MARKER@
+Exec=eric6_webbrowser %u@MARKER@
 MimeType=text/html;text/xml;application/xhtml+xml;x-scheme-handler/http;x-scheme-handler/https;
 Icon=ericWeb@MARKER@
 Terminal=false
